R/ggparcoord.R

```diff
@@ -309,9 +309,18 @@ ggparcoord <- function(
   inner_rescaler <- function(x, type = "sd", ...) {
     # copied directly from reshape because of import difficulties :-(
     # rescaler.data.frame
-    continuous <- sapply(x, is.numeric)
+    continuous    <- sapply(x, is.numeric)
     if (any(continuous)) {
-      x[continuous] <- lapply(x[continuous], inner_rescaler_default, type = type, ...)
+      if (type %in% c("sd", "robust", "range")) {
+        singleVal    <- sapply(x, function(col){if(length(unique(col)) == 1) TRUE else FALSE}) # indicating columns containing only one single value
+        isFactor     <- as.vector(sapply(x, is.factor))
```

Comment:
You are right. I removed this in a new commit. I guess it was left over when I played with columns storing a single factor. I didn't meant to litter the code.